static/src/javascripts/projects/common/modules/spacefinder.js

```diff
@@ -173,6 +186,7 @@ const enforceRules = (measurements, rules, exclusions) => {
 
 	// enforce selector rules
 	if (rules.selectors) {
+		let selectorExclusions = [];
```

Comment:
Can this be a `const` here? I understand that the contents of the array are being mutated, but the reference of the array itself is never reassigned?

Reply:
Yup you're right! Have fixed